judgels-frontends/raphael/src/routes/uriel/contests/single/supervisors/ContestSupervisorsTable/ContestSupervisorsTable.tsx

```diff
@@ -45,6 +60,12 @@ export class ContestSupervisorsTable extends React.PureComponent<ContestSupervis
       <tr key={supervisor.userJid}>
         <td>
           <UserRef profile={profilesMap[supervisor.userJid]} />
+          {supervisor.managementPermissions &&
```

Comment:
Let's put this in a separate column (`<td/>`), with title "Management permissions"